### PR TITLE
fix: added '_redirect' file

### PR DESCRIPTION
**Pull request by shamimbinnur**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR resolves the "Page not found problem" while accessing any route with a direct link. 

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Review by 0-vortex — CHANGES_REQUESTED** [review]
Please move this to the `netlify.toml` configuration file available in the root of the project:
- all the netlify configuration is stored there
- adding the redirects file to public has additional side effects

Edit links:
- https://github.com/open-sauced/hot/blob/main/netlify.toml
- https://docs.netlify.com/routing/redirects/

**Comment by bdougie** [comment]
@shamimbinnur checking in on this to see if you can swap this to the toml

```
[[redirects]]
  from = "/*"
  to = "/index.html "
  status = 200
```

**Comment by shamimbinnur** [comment]
> @shamimbinnur checking in on this to see if you can swap this to the toml
> 
> ```
> [[redirects]]
>   from = "/*"
>   to = "/index.html "
>   status = 200
> ```

@bdougie added this in toml file

File: netlify.toml [hunk 1]
```diff
@@ -4,3 +4,8 @@ command = "npm run build"
 
 [context.beta]
 environment = { CHANNEL = "beta" }
+
+[[redirects]]
+  from = "/*"
+  to = "/index.html "
```
Comment (0-vortex) [inline netlify.toml:10]: Please remove the space between `html` and `"`

**Review by 0-vortex — APPROVED** [review]
LGTM, merging this on beta!

**Pull request merged**